content/developer/reference/orm.rst

```diff
@@ -340,16 +340,17 @@ if all fields in the sequence are required), ``groups``, ``digits``, ``size``,
 ``context``. All semantic-free attributes are copied from the source
 field.
 
-Until specified explicitly, related field has ``readonly=True`` and
-``copy=False``.
+By default, related fields are:
```

Comment:
I thought it was better to specify all the defaults for related fields together (and fix the misleading note).
If it's ok for you, I'll squash the commit and merge it ;) (while keeping your authorship ofc).

Reply:
Looks good :+1: